homeassistant/components/websocket_api/commands.py

```diff
@@ -723,3 +724,17 @@ async def handle_supported_brands(
             raise int_or_exc
         data[int_or_exc.domain] = int_or_exc.manifest["supported_brands"]
     connection.send_result(msg["id"], data)
+
+
+@callback
+@decorators.websocket_command(
+    {
+        vol.Required("type"): "supported_features",
+        vol.Required("features"): dict,
```

Comment:
Let's type it. Should supported feature be int or float? 
```suggestion
        vol.Required("features"): {str: int},
```

Reply:
I don't think we would ever overflow this so int is probably fine